### ISSUE TITLE
deafing a power switch to the robot and arranging the arduino Day #32

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#58]
**Today, May 6, 2024, we were thinking about mounting a switch on the robot so we wouldn't have to disconnect the battery to turn off the robot.**

- The goal today is to mount a switch on the robot to turn the robot on and off.

**Comment by WROMagnet** [comment]
We will be arranging the arduino Uno under the Prizm, we will place the prizm and below it will be the arduino UNO. We already assembled it and here with a photo of the arduino below the arduino.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/ae0a331a-760f-45e9-9ec3-88690869e92b)

Since we accommodated the Arduino UNO, now @LuisriosWRO3 will be assembling the switch. Here we have a video of @LuisriosWRO3 deafening the switch cables.

https://github.com/WROMagnet/Magnet-Robots/assets/166937907/145a5da0-9f4b-4082-9dcb-70bb61674899

Here is the result of the Switch



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/2c14e357-5432-49f6-8b2f-b18adffe8e99)
